com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/SetBreakpointsRequestHandler.java

```diff
@@ -86,7 +86,15 @@ public void initialize(IDebugAdapterContext context) {
         if (context.getDebugSession() == null) {
             return AdapterUtils.createAsyncErrorResponse(response, ErrorCode.EMPTY_DEBUG_SESSION, "Empty debug session.");
         }
-
```

Comment:
1. If don't handle these requests(breakpoint/threads, etcs), we should either not register handlers in the DebugAdapter or just pass a DummyRequestHandlers.. 
2. There are requests more than threads/breakpoints should also be handled.